HHS/Head-Start-TTADP
# [TTAHUB-2315] Cache IST/NC on completed training reports

## Description of change
- Cache the user name and national center name for completed training reports (add new table + migration)
- Display the national center name in the collaborators dropdown
- Add a script to run to fill in missing cached data for reports already completed (can be removed once run)

## How to test
- Run ```updateCompletedEventReportPilots``` on prod like data and confirm that the script runs and completed training reports
-  Confirm that the collaborators dropdown shows the national center name
- Save a training report and confirm that the new table ```EventReportPilotNationalCenterUsers``` is updated
- View a completed training report and confirm that the information displayed there (as far as user/national center) is correct

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2315


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
